applications/IgaApplication/custom_conditions/sbm_laplacian_condition.cpp

```diff
@@ -348,17 +348,17 @@ void SBMLaplacianCondition::CalculateAllNeumann(
         Vector projection(3);
         projection = candidate_closest_skin_segment_1.GetGeometry()[closestNodeId].Coordinates() ;
 
-        Vector d(3);
-        noalias(d) = projection - r_geometry.Center().Coordinates();
+        Vector mDistanceVector(3);
+        noalias(mDistanceVector) = projection - r_geometry.Center().Coordinates();
 
         const Matrix& N = r_geometry.ShapeFunctionsValues();
 
         noalias(DN_DX) = r_DN_De[0]; // prod(r_DN_De[point_number],InvJ0);
 
         // Compute the normal
-        array_1d<double, 3> normal_parameter_space;
-        normal_parameter_space = - r_geometry.Normal(0, GetIntegrationMethod());
-        normal_parameter_space = normal_parameter_space / MathUtils<double>::Norm(normal_parameter_space);
+        array_1d<double, 3> mNormalParameterSpace;

```

Comment:
This is strange, if `mNormalParameterSpace` is a member variable this would be a compile error...